### ISSUE TITLE
Не устанавливается окружение в windows 10

### ISSUE THREAD

**Issue by MrLamberg** [SemyonSinchenko/qmlcourse#199]
Первоначально выполнял установку окружения как написано в инструкции

> PS C:\Users\user> cd .\qmlcourse\
> PS C:\Users\user\qmlcourse> make install-windows
> cmd //C curl https://www.python.org/ftp/python/3.8.5/python-3.8.5.exe --output "%TMP%\python-3.8.5.exe" && "%TMP%\python-3.8.5.exe" /silent
> cmd //C curl https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py --output "%TMP%\get-poetry.py"
> cmd //C python3 -V
> cmd //C python3 "%TMP%\get-poetry.py"
> cmd //C "C:\tools\miniconda3\Scripts\conda install psi4=1.4rc4.dev1 python=3.8 -c psi4/label/dev -c conda-forge"
> cmd //C "%USERPROFILE%\.poetry\bin\poetry remove tensorflow-quantum"
> cmd //C "%USERPROFILE%\.poetry\bin\poetry install"
> Microsoft Windows [Version 10.0.18363.1556]
> (c) Корпорация Майкрософт (Microsoft Corporation), 2019. Все права защищены.
> 
> C:\Users\user\qmlcourse>make build-windows
> make[1]: Entering directory 'C:/Users/user/qmlcourse'
> cmd //C "%USERPROFILE%\.poetry\bin\poetry run psi4 --test"
> cmd //C "%USERPROFILE%\.poetry\bin\poetry run jupyter-book build ./qmlcourseRU"
> Microsoft Windows [Version 10.0.18363.1556]
> (c) Корпорация Майкрософт (Microsoft Corporation), 2019. Все права защищены.

Соответственно ничего не поставилось.
Заметил что команды cmd //C в windows нет.
Поменял на cmd /C и получил другую ошибку 

> C:\Users\user\qmlcourse>make install-windows
> cmd /C curl https://www.python.org/ftp/python/3.8.5/python-3.8.5.exe --output "%TMP%\python-3.8.5.exe" && "%TMP%\python-3.8.5.exe" /silent
> cmd /C curl https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py --output "%TMP%\get-poetry.py"
> cmd /C python3 -V
> cmd /C python3 "%TMP%\get-poetry.py"
> cmd /C "C:\tools\miniconda3\Scripts\conda install psi4=1.4rc4.dev1 python=3.8 -c psi4/label/dev -c conda-forge"
> cmd /C "%USERPROFILE%\.poetry\bin\poetry remove tensorflow-quantum"
> cmd /C "%USERPROFILE%\.poetry\bin\poetry install"
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100 25.5M  100 25.5M    0     0  25.5M      0  0:00:01 --:--:--  0:00:01 29.1M
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100 32677  100 32677    0     0  32677      0  0:00:01 --:--:--  0:00:01 80485
> Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.
> Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.
> Collecting package metadata (current_repodata.json): failed
> 
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/psi4/label/dev/win-64/current_repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> 'https://conda.anaconda.org/psi4/label/dev/win-64'
> 
> 
> Системе не удается найти указанный путь.
> Системе не удается найти указанный путь.
> make: *** [Makefile:37: install-python-poetry-windows] Error 1

**Comment by MrLamberg** [comment]
делал все по инструкции через powershell с этой страницы https://semyonsinchenko.github.io/qmlcourse/_build/html/book/pythonblock/python_l2.html#windows

**Comment by SemyonSinchenko** [comment]
Значит это косяк на нашей стороне.

**Comment by dkorzhov** [comment]
@MrLamberg можешь попробовать сделать теже команды, только с cmd /C .........? То есть у себя локально удалить один "/". Что пишет при этом?

**Comment by MrLamberg** [comment]
Код со страницы не работает
ошибка 

> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> ~\AppData\Local\Temp/ipykernel_11104/693153482.py in <module>
> ----> 1 import psi4
>       2 
>       3 h_atom = psi4.geometry("H")
> 
> ModuleNotFoundError: No module named 'psi4'

При выполнении команды тоже ошибка
> PS C:\Users\user\qmlcourse> cmd /C "C:\tools\miniconda3\Scripts\conda install psi4=1.4rc4.dev1 python=3.8 -c psi4/label/dev -c conda-forge"
> Collecting package metadata (current_repodata.json): failed
> 
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/psi4/label/dev/win-64/current_repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> 'https://conda.anaconda.org/psi4/label/dev/win-64'

**Comment by dkorzhov** [comment]
> ondaHTTPError: HTTP 000 CONNECTION FAILED for url

попробуй сначала запустить conda init, затем команду conda.....

**Comment by vtrokhymenko** [comment]
> @SemyonSinchenko хорошо бы еще на MacOS прогнать установку. Когда прогонял на GitHub там уже стоял python3.8 и поэтому я его не включал в базовую сборку. Может или в инструкции надо пояснить, чтобы сами ставили или добавить, но при этом проверить на стационарной OS.

да, нужно проверить. @stalkermustang можешь протестировать и дописать чего не хватает?

**Comment by MrLamberg** [comment]
Это не виртуалка.
Выполнил команды, ничего не изменилось 

> PS C:\users\user\qmlcourse> cmd "C:\tools\miniconda3\Scripts\conda install psi4=1.4rc4.dev1 python=3.8 -c psi4/label/dev -c conda-forge"
> Microsoft Windows [Version 10.0.18363.1556]
> (c) Корпорация Майкрософт (Microsoft Corporation), 2019. Все права защищены.
> 
> C:\users\user\qmlcourse>cmd /C "C:\tools\miniconda3\Scripts\conda install psi4=1.4rc4.dev1 python=3.8 -c psi4/label/dev -c conda-forge"
> Collecting package metadata (current_repodata.json): failed
> 
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/psi4/label/dev/win-64/current_repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> 'https://conda.anaconda.org/psi4/label/dev/win-64'
